src/_posts/languages/ruby/rails/2000-01-01-start.md

```diff
@@ -173,3 +173,10 @@ git add bin/*
 git commit -m "Add Yarn and Webpacker binstubs"
 git push scalingo master
 ```
+
+## Use Vips to process image
```

Comment:
I though about this (as I know you handle this carefully) and Rails is writing `Vips` https://edgeguides.rubyonrails.org/active_storage_overview.html#transforming-images

Reply:
Do you still prefer VIPS over Vips ? No preference for me

Reply:
> Do you still prefer VIPS over Vips ? No preference for me

Then no preference either for me :)

> Do you think we should handle this at the app level (via Ruby or CSS) ? Or is it better that it's done in the .md file ?

I didn't know it could be handled at the CSS level. It's also a suitable solution for me. No preference.